include/ps/kv_app.h

```diff
@@ -88,7 +88,8 @@ class KVWorker : public SimpleApp {
 #ifdef USE_PROFILING
     const char *val;
     val = Environment::Get()->find("DMLC_ENABLE_RDMA");
-    is_worker_zpull_ = val ? atoi(val) : false;
+    is_worker_zpull_ = val ? true : false;
```

Comment:
I am afraid that if someone set `DMLC_ENABLE_RDMA=0`, `is_worker_zpull_` is still true. This could affect the correctness of ZMQ.

Reply:
Good catch, I changed the env var at the last minute. Let me fix it